templates/web/components/bmviii_footer.tmpl

```diff
@@ -0,0 +1,49 @@
+{{ define "bmviii footer" }}
+
+		<footer class="footer is-fixed-bottom" style='background-color: #54A0EE;'>
+			<div class="container">
+				<div class="columns is-centered is-mobile">
+				    <div class="column is-3">
+                        <img alt="Social" src={{ static_path "bmviii_socials_text.svg" }}>
+                        <div class="columns is-gapless is-mobile" style="padding-top: 10px">
+                            <div class="column is-4-mobile is-3-tablet is-2-desktop">
+                                <p>
+                                    <a href="https://www.facebook.com/BoilerMake/">
+                                        <img alt="Facebook logo" src={{ static_path "bmviii_facebook.svg" }} style="padding-right: 2px">
+                                    </a>
+                                </p>
+                            </div>
+                            <div class="column is-4-mobile is-3-tablet is-2-desktop">
+                                <p>
+                                    <a href="https://www.instagram.com/boilermake/">
+                                        <img alt="Instagram logo" src={{ static_path "bmviii_instagram.svg" }} style="padding-right: 2px">
+                                    </a>
+                                </p>
+                            </div>
+                            <div class="column is-4-mobile is-3-tablet is-2-desktop">
+                                <p>
+                                    <a href="https://twitter.com/BoilerMake1/">
+                                        <img alt="Twitter logo" src={{ static_path "bmviii_twitter.svg" }}>
+                                    </a>
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+					<div class="column is-3 is-offset-1">
+						<img alt="Pages" src={{ static_path "bmviii_pages_text.svg" }}>
+						<div style="padding-left: 10px">
```

Comment:
Is the padding-left here intentional?
![Screen Shot 2020-10-30 at 21 11 19](https://user-images.githubusercontent.com/10781907/97767730-13b27a80-1aec-11eb-82ef-7b1cd4bc15ec.png)

Reply:
Yeah, cause on the spec it looks like its centered

Reply:
Oh then I'd try to actually center/middle align the text this looks kinda janky

Reply:
Actually Ima see if I can just leave to the left. Center aligning it won't really work cause the div isn't the same size as the image

Reply:
Or maybe just having the centering on desktop sizes